### DISCUSSION TITLE
Change layout of "Users in Role"

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#8764] [Feature Requests]
So the Users in Role section of the Roles & Permissions page is fine for about maybe 30 users, but if you have 100s of users the page will be extremely long. So I suggest a flyout modal. That will have a table with pagination holding the user records (also a search filter).

![Screen Shot 2021-10-13 at 3 44 08 PM](https://user-images.githubusercontent.com/9326922/137156497-b43406ec-c924-4612-8bb1-6578e24b3eb9.jpg)

**Comment by rijkvanzanten** [comment]
Ref https://github.com/directus/directus/issues/3889

**Comment by br41nslug** [comment]
has been implemented
